authors/apps/articles/views.py

```diff
@@ -61,6 +61,18 @@ class RetrieveUpdateDestroyArticle(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = serializers.ArticleSerializer
     lookup_field = 'slug'
 
+    def get_serializer_context(self):
```

Comment:
@ezrogha please try removing this method/function to see if the tests can pass without overriding it. I feel the default unridden implementation will work as well.

Reply:
You are right @ja-odur, tests are actually passing